components/sdmmc/sdmmc_cmd.c

```diff
@@ -191,6 +196,17 @@ esp_err_t sdmmc_card_init(const sdmmc_host_t* config, sdmmc_card_t* card)
 
         /* Send SEND_OP_COND (ACMD41) command to the card until it becomes ready. */
         err = sdmmc_send_cmd_send_op_cond(card, host_ocr, &card->ocr);
+		
+		//if time-out try switching from SD to MMC and vice-versa
```

Comment:
Please convert indentation to be the same as surrounding code

Reply:
done